yarn.lock

```diff
@@ -18200,7 +18200,7 @@ string-length@^5.0.1:
     char-regex "^2.0.0"
     strip-ansi "^7.0.1"
 
-"string-width-cjs@npm:string-width@^4.2.0":
+"string-width-cjs@npm:string-width@^4.2.0", string-width@^4.1.0, string-width@^4.2.0, string-width@^4.2.3:
```

Comment:
Is it possible for you to revert the `yarn.lock` change? I think it might actually be incorrect for this line.